point_viewer_proto_rust/src/proto.proto

```diff
@@ -75,23 +75,50 @@ message NodeId {
   int64 deprecated_index = 2;
 }
 
-message Node {
-  enum PositionEncoding {
+enum PositionEncoding {
     INVALID = 0;
     Uint8 = 1;
     Uint16 = 2;
     Float32 = 3;
     Float64 = 4;
-  }
+}
 
+
+message Node {
   PositionEncoding position_encoding = 2;
   int64 num_points = 3;
   NodeId id = 4;
 }
 
+enum AttributeData{
```

Comment:
How about `AttributeType` or `AttributeDataType`?

Reply:
`enum AttributeData{` -> `enum AttributeData {`

Reply:
I thought I'd use proto::AttributeData -> AttributeData like proto::PositionEncoding.. any thoughts?

Reply:
The difference is that `AttributeData` really contains data, while this `enum` here only describes the data type it contains.